packages/delayedInbox-l2msg/README.md

```diff
@@ -2,7 +2,9 @@
 (Noted this can only be done in nitro stack, not in arbitrum classic, so l2 rpc please use nitro test rpc)
 delayedInbox-l2msg is a simple sample example that allows you to send a l2 msg without using sequencer way, this can be used when sequencer censor your tx or when sequencer downs.
 
-The demo will alow you send a l2 message while you don't need to use l2 rpc but just use l1 rpc; This demo has 2 part, one part will show how to send a normal l2 transaction using delayed inbox, another will show when sequencer downs, how to withdraw your funds back.
+The demo will alow you send a l2 message while you don't need to use l2 rpc but just use l1 rpc; This demo has 2 part, one part will show how to send a normal l2 transaction using delayed inbox, another will show how to withdraw your funds back without sequencer.
```

Comment:
The demo will allow you to send a l2 message while you don't need to use l2 rpc but just use l1 rpc; This demo has 2 parts; one part will show how to send a normal l2 transaction using delayed inbox, another will show how to withdraw your funds back without sequencer.